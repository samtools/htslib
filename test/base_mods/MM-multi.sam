@CO	Testing multiple m, h and N modifications on the same read.
@CO	r1 has them separated out.
@CO	r2 has them combined together, for example as produced by
@CO	a joint basecaller which assigns probabilities to all
@CO	trained events simultaneously.
r1	0	*	0	0	*	*	0	0	AGCTCTCCAGAGTCGNACGCCATYCGCGCGCCACCA	*	Mm:Z:C+m,2,2,1,4,1;C+h,6,7;N+n,15,2;	Ml:B:C,128,153,179,204,230,159,6,215,240	MN:i:36
r2	0	*	0	0	*	*	0	0	AGCTCTCCAGAGTCGNACGCCATYCGCGCGCCACCA	*	Mm:Z:C+mh,2,2,0,0,4,1;N+n,15;	Ml:B:C,77,159,103,133,128,108,154,82,179,57,204,31,240
