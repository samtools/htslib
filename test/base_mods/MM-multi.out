0	A
1	G
2	C
3	T
4	C
5	T
6	C	C+m128
7	C
8	A
9	G
10	A
11	G
12	T
13	C
14	G
15	N	N+n215
16	A
17	C	C+m153
18	G	N+n240
19	C	C+h159
20	C	C+m179
21	A
22	T
23	Y
24	C
25	G
26	C
27	G
28	C
29	G
30	C
31	C	C+m204
32	A
33	C
34	C	C+m230 C+h6
35	A
---
Present: m. h. n.
6	C	C+m128
15	N	N+n215
17	C	C+m153
18	G	N+n240
19	C	C+h159
20	C	C+m179
31	C	C+m204
34	C	C+m230 C+h6

===

0	A
1	G
2	C
3	T
4	C
5	T
6	C	C+m77 C+h159
7	C
8	A
9	G
10	A
11	G
12	T
13	C
14	G
15	N	N+n240
16	A
17	C	C+m103 C+h133
18	G
19	C	C+m128 C+h108
20	C	C+m154 C+h82
21	A
22	T
23	Y
24	C
25	G
26	C
27	G
28	C
29	G
30	C
31	C	C+m179 C+h57
32	A
33	C
34	C	C+m204 C+h31
35	A
---
Present: m. h. n.
6	C	C+m77 C+h159
15	N	N+n240
17	C	C+m103 C+h133
19	C	C+m128 C+h108
20	C	C+m154 C+h82
31	C	C+m179 C+h57
34	C	C+m204 C+h31

===

